### ISSUE TITLE
Add a "World" option to the Eco-Score dashboard dropdown

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/hunger-games#639]
### Story
- Add a "World" option to the Eco-Score dashboard dropdown
![image](https://github.com/openfoodfacts/hunger-games/assets/1689815/c5986fdc-6d13-4318-a1d4-05b040507972)

### Part of
- #103

**Comment by Jittojoyes98** [comment]
Can i work on this? Iam new here.

**Comment by alexfauquette** [comment]
Yes, feel free to ask questions if you are blocked 👍

**Comment by Jittojoyes98** [comment]
Do you mean the Country dropdown in the Dashboard ?

**Comment by VaiTon** [comment]
> Do you mean the Country dropdown in the Dashboard ?

Nope, the dropdown here: https://hunger.openfoodfacts.org/eco-score

**Comment by teolemon** [comment]
I'm bumping this one to P0
